extensions/rich_text_components/Skillreview/protractor.js

```diff
@@ -18,15 +18,25 @@
  */
 
 var objects = require('../../objects/protractor.js');
+var forms = require(
+  '../../../core/tests/protractor_utils/forms.js');
 
-var customizeComponent = function(modal, index) {
+var customizeComponent = function(modal, text) {
+  objects.UnicodeStringEditor(
+    modal.element(by.tagName('schema-based-unicode-editor'))
+  ).setValue(text);
   objects.SkillSelector(
     modal.element(by.tagName('skill-selector-editor'))
-  ).setValue(index);
+  ).setValue();
```

Comment:
I don't understand this test utility. Don't you need to set this value to an actual skill that gets passed in as an arg to the customization function?

Reply:
Since this is just for a test, I made it choose the first skill in the list always and so there is no need to pass an index inside, is that ok?

Reply:
Probably not, actually. You're building general test utility functionality here, and other developers (including future-you!) will probably need to select specific skills in the future. The test utility method should therefore reflect what actions are available to the user, so I would recommend handling the argument correctly here (you could select by the skill name perhaps).

Reply:
Done